selfhosting/auto-scaling-aws.md

```diff
@@ -0,0 +1,91 @@
+# AWS EKS auto scaling instructions
+
+Unlinke GCP, AWS EKS doesn't enable autoscaling by default. You need to install
+[autoscaler][k8s-autoscaler] specific to AWS EKS cluster. Thanks to helm, we
+can install cluster-autoscaler chart. First prepare iam roles and node groups
+as suggested in readme of [helm chart][helm-cluster-autoscaler]. Steps to
+follow:
```

Comment:
> First prepare iam roles and node groups
 as suggested in readme of [helm chart][helm-cluster-autoscaler]

Does this line imply that user has to manually read the readme and do the steps mentioned there ? I think not.

Reply:
Oh yeah, no. Need to change this.